src/lib.rs

```diff
@@ -1149,15 +1078,14 @@ mod tests {
         let green_hello = "\u{1b}[0m\u{1b}[32mHello\u{1b}[0m";
         let blue_world = "\u{1b}[0m\u{1b}[34mWorld!\u{1b}[0m";
         assert_eq!(
-            fill(&(String::from(green_hello) + " " + &blue_world), 6),
-            String::from(green_hello) + "\n" + &blue_world
+            fill(&format!("{} {}", green_hello, blue_world), 6),
```

Comment:
While this is a nice change, it seems unrelated. It would be great to have this as a esparate PR.